### DISCUSSION TITLE
Audiobook Metadata Agent dosen't seem to find any matches in Plex

### DISCUSSION THREAD

**Discussion by Dokixi** [seanap/Plex-Audiobook-Guide#31] [Q&A] [answered]
I have setup the Audiobook.bundle folder into Plex and done everything right in the Plex settings setup etc. according to the guide. So all things should work.

But it dosen't seem like I get any matches with the Audiobook agent for some reason. Even if I try any other combination of names and removing the year in the repair match window in Plex.

What am I doing wrong? 🤔

**Comment by rabelux** [comment, marked as answer]
Have a look at https://github.com/seanap/Audiobooks.bundle/issues/20.
@djdembeck recently did a complete rewrite of the plugin and apparently that needs some further testing.

**Reply by Dokixi** [reply]
Oh, okay. I just started to use this audiobook metadata agent with Plex after going through seanap's audiobook guide, and seeing this issue being this recent, I guess I was quite unlucky with the timing 😅 Glad to see it's atleast being worked on and hopefully fixed very soon.

**Reply by seanap** [reply]
Latest PMS version 1.24.4 fixes the issue.  Checkout the new agent here https://github.com/djdembeck/Audnexus.bundle

**Reply by rabelux** [reply]
There is very little information about the new plugin. I guess it does what the current plugin does but uses different sources? If that's the case, what sources does it use?

**Reply by djdembeck** [reply]
I will be writing up some documentation on it later today. It's purpose is to be API only with no scraping. Only US region is supported currently.
